src/shared/components/LomFieldsInput/LomFieldsInput.helpers.ts

```diff
@@ -19,9 +20,15 @@ export const mapOptionsToLomFields = (
 	});
 };
 
-export const getParentContext = (loms: Avo.Lom.LomField[], allLoms: Avo.Lom.LomField[]) => {
+export const getParentContext = (
```

Comment:
Let's use education level instead of context, since it is less confusing
```suggestion
export const getParentEducationLevel = (
```